tests/lts_compatibility.py

```diff
@@ -236,6 +237,18 @@ def run_code_upgrade_from(
                 network.retire_node(primary, node)
                 if primary == node:
                     primary, _ = network.wait_for_new_primary(primary)
+                    spec = inspect.signature(network.txs.issue)
+                    if "record_claim" in spec.parameters:
+                        LOG.info("Run transaction with additional claim")
+                        txid = network.txs.issue(
+                            network, number_txs=1, record_claim=True
+                        )
+                        LOG.info(
+                            "Check receipts are fine, including transaction with claims"
+                        )
+                        test_random_receipts(
+                            network, args, lts=True, additional=[txid.seqno]
```

Comment:
Could you rename `additional` to `additional_seqnos` or something similar? I (wrongly) first assumed that additional referred to additional claims in the receipt.

Reply:
Done